packages/@tinacms/graphql/src/schema/createSchema.ts

```diff
@@ -10,20 +10,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import { TinaSchema, TinaCloudSchemaBase } from '@tinacms/schema-tools'
+import { TinaSchema } from '@tinacms/schema-tools'
+import type { Schema } from '@tinacms/schema-tools/dist/types'
+import type { Schema as SchemaWithNamespace } from '@tinacms/schema-tools'
 
-import { validateSchema } from './validate'
 // @ts-ignore File '...' is not under 'rootDir'
 import packageJSON from '../../package.json'
 
 export const createSchema = async ({
   schema,
   flags = [],
 }: {
-  schema: TinaCloudSchemaBase
+  schema: Schema | SchemaWithNamespace
   flags?: string[]
 }) => {
-  const validSchema = await validateSchema(schema)
```

Comment:
There's another `validateSchema` function in `@tinacms/schema-tools`, and it's getting called from `defineConfig`, so I don't think this is necessary anymore